base_layer/core/src/proof_of_work/monero_rx/helpers.rs

```diff
@@ -110,9 +127,9 @@ pub fn verify_header(header: &BlockHeader) -> Result<MoneroPowData, MergeMineErr
 }
 
 /// Extracts the Monero block hash from the coinbase transaction's extra field
-pub fn extract_tari_hash(monero: &monero::Block) -> Result<Option<monero::Hash>, MergeMineError> {
-    let extra_field = ExtraField::try_parse(&monero.miner_tx.prefix.extra)
-        .map_err(|_| MergeMineError::DeserializeError("Invalid extra field".to_string()))?;
+pub fn proxy_extract_tari_hash_from_block(monero: &monero::Block) -> Result<Option<monero::Hash>, MergeMineError> {
```

Comment:
```suggestion
pub fn extract_tari_hash_from_block_truncate(monero: &monero::Block) -> Result<Option<monero::Hash>, MergeMineError> {
```
Using the proxy name here is adding confusion here. reading the name I cannot infer what proxy does, or what makes it unique. Having knowledge of the complete codebase and knowing exactly what makes it different in the monero methods, I can infer what makes it special. But we need to ensure that just reading this code we can infer the method meanings,